### ISSUE TITLE
Precompiling JuMP

### ISSUE THREAD

**Issue by mlubin** [JuliaOpt/JuMP.jl#329]
It works:

Before:
```
$ julia -e "tic(); using JuMP; toc()"
elapsed time: 4.546343258 seconds
```

After:
```
$ julia -e "tic(); using JuMP; toc()"
elapsed time: 0.010020969 seconds
```

Just need to add:
```
Base.require("MathProgBase")
Base.require("JuMP")
```
to julia/base/userimg.jl and re-run ``make``.

Opening this for discussion, not sure if we should publicize it yet.

**Comment by IainNZ** [comment]
Works beautifully - I put Gurobi in for good measure, that works too, or so it seems.

**Comment by mlubin** [comment]
Closed by #515
